ListKit.xcodeproj/project.pbxproj

```diff
@@ -389,21 +389,24 @@
 			attributes = {
 				LastSwiftMigration = 0700;
 				LastSwiftUpdateCheck = 0700;
-				LastUpgradeCheck = 0700;
+				LastUpgradeCheck = 0800;
```

Comment:
It seems like you're on Xcode 8, not 8.1?

If that's the case, would you mind to update this PR with Xcode 8.1 which has Swift 3.0.1 support? 😄